docs/samples/custom/torchserve/docs/canary.md

```diff
@@ -6,11 +6,12 @@
 apiVersion: "serving.kubeflow.org/v1beta1"
 kind: "InferenceService"
 metadata:
- name: "torchserve-custom"
+  name: "torchserve-custom"
 spec:
- predictor:
-   pytorch:
-     storageUri: "pvc://model-pv-claim"
+  predictor:
+    containers:
+    - image: {username}/torchserve:latest
```

Comment:
@jagadeeshi2i Please check, previous version had the storageUri steps

Reply:
We do it by changing the image in custom server without external storage and the storageUri for custom server with external storage.